docs/topics/vscode-extension-resolving-issues.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-resolving-issues_{context}']
+= Resolving issues
+
+You can resolve issues by using a _Quick Fix_ automatic code replacement or by editing the code. You can use the *Delete* and *Mark as Complete* options to keep track of which issues you have finished working with. Files marked deleted or completed will be analyzed again the next time you analyze a project that contains them.
+
+.Procedure
+
+. Review the issues in the left pane and do one of the following actions:
+
+* If an issue displays the Quick Fix icon (image:vs_optional.png[Optional or Quick Fix]):
```

Comment:
The icon is a bit too large and is positioned very high. You can crop empty space around the icon and play with the image size setting.